Tests/Unit/Module/STIG.DomainName.tests.ps1

```diff
@@ -0,0 +1,92 @@
+#region Header
+. $PSScriptRoot\.tests.header.ps1
+#endregion
+
+$domainName = 'Domain.test'
+$forestName = 'Forest.test'
+$domainDistinguished = "DC={0},DC={1}" -f $domainName.Split(".")
+$forestDistinguished = "DC={0},DC={1}" -f $forestName.Split(".")
+$domainDistinguishedPart1 = "DC=Domain"
+$domainDistinguishedPart2 = "DC=Domain,DC=Test"
+$parts = @("Domain","Test")
+
+Describe 'Get-DomainName' {
```

Comment:
All of the It statements in this describe block are indented one too many times, should have a single indent, appears to be double.

Reply:
Good catch. Updated